finish/tutorial-lint/proto/acme/weather/v1/weather.proto

```diff
@@ -3,12 +3,11 @@ syntax = "proto3";
 package acme.weather.v1;
 
 enum Condition {
-  CONDITION_UNSPECIFIED = 0;
-  CONDITION_SUNNY = 1;
-  CONDITION_RAINY = 2;
+  SUNNY = 0;
+  RAINY = 1;
 }
 
-message GetWeatherRequest {
```

Comment:
Why did this change? Our recommendation is for request types to be named like this.